### PR TITLE
Remove vendored six with much smaller compat library

### PR DESCRIPTION
**Pull request by beniwohli**
This replaces six with a much smaller `compat` library that only contains a small subset of six

### CODE REVIEW

File: elasticapm/contrib/async_worker.py [hunk 1]
```diff
@@ -49,23 +49,20 @@ def main_thread_terminated(self):
...
+                    "PID %i: ElasticAPM is attempting to send %i pending messages\n" % (
                         os.getpid(),
                         size,
                     )
                 )
-                six.print_(
-                    "Waiting up to %s seconds, " % ELASTIC_APM_WAIT_SECONDS,
-                    end=''
-                )
+                sys.stdout.write("Waiting up to %s seconds, " % ELASTIC_APM_WAIT_SECONDS)
```
Comment (ruflin) [inline elasticapm/contrib/async_worker.py]: Did you leave out the newline here on purpose?

Comment (beniwohli) [reply elasticapm/contrib/async_worker.py]: Yes. `six.print_('...', end='')` is equivalent with not writing a newline (same with Python 3's print function).

**Comment by beniwohli** [comment]
@ruflin squashed :)

**Comment by beniwohli** [comment]
@ruflin IGFH (It's Green For Hours) ;p

**Comment by ruflin** [comment]
@beniwohli SATWBM (sorry about that was blind, merged)

**Pull request merged**